compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/BIRPackageSymbolEnter.java

```diff
@@ -330,9 +340,15 @@ private void defineImportPackage(DataInputStream dataInStream) throws IOExceptio
     }
 
     private void defineFunction(DataInputStream dataInStream) throws IOException {
+        dataInStream.readInt(); // skip line start
+        dataInStream.readInt(); // skip line end
+        dataInStream.readInt(); // skip col start
+        dataInStream.readInt(); // skip col end
```

Comment:
We can call the skipPosition method here if we pass it a parameter. `skipPosition(dataInStream, 4);` we can add the comments above the call if it makes sense